.github/workflows/compose.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
I would suggest changing the name of this file as vs code thinks it is a docker-compose file and messes up linting